resources/META-INF/plugin.xml

```diff
@@ -219,6 +219,13 @@
                          enabledByDefault="true" level="WARNING"
                          implementationClass="com.magento.idea.magento2plugin.inspections.xml.InvalidDependencyInjectionTypeInspection"/>
 
+        <localInspection language="XML" groupPath="XML"
+                         shortName="PreferenceXmlInspections"
+                         displayName="Inspection the existence of PHP classes for attributes in the preference"
```

Comment:
@Iamwade, please extract displayName attribute value to the magento2.inspection bundle and reference it from this context:
```suggestion
                         bundle="magento2.inspection" key="inspection.displayName.PreferenceXmlInspections"
```

Reply:
Fixed.

Reply:
Thank you!